brian_cheung/week_one_challenges/string_manipulation.md

```diff
@@ -19,13 +19,27 @@ iverson_rant = "If I can't practice, I can't practice, man. If I'm hurt, I'm hur
 Programmatically manipulate `iverson_rant` to reflect the output above.
 
 ```ruby
-# write your code here
+def simplify (phrase)
+  abbr = nil
+  wordarray = phrase.split
+    wordarray.each do |word|
+      addabbr = "#{word}".chr
+      abbr = "#{abbr}#{addabbr}"
+    end
+  puts "#{abbr}"
+end
 ```
```

Comment:
Some more nitpicks:
* `wordarray` should have an underscore for readability: `word_array`
* Generally, you want to name your variables as specifically as possibly, for readability's sake. What the heck is `addabbr`? Something like `first_letter` might be more appropriate.
* The only time you need to interpolate is on line 27, when you're combining two strings together. `word.chr` and `puts abbr` don't need interpolation - they are already strings.

Check your other methods for these same nitpicks 😄